deployments/kubernetes/compute.test/kustomization.yaml

```diff
@@ -143,3 +135,21 @@ patches:
       name: sdn-controller
       namespace: merak
 
+# # Temporal Helm Chart
+# helmCharts:
+# - releaseName: temporal
+#   repo: https://charts.wener.tech
+#   name: temporal
+#   version: 0.15.1
+#   valuesInline:
+#     server:
+#       replicatCount: 1
+#     cassandra:
```

Comment:
@phudtran can you put a README file under `compute.test` folder to describe what those files for?
Also, if the commented-out lines in the file are not needed, please remove them.

Reply:
It has a README, I'll remove this commented out chart.